File: 1/solution.py
from typing import List


class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        tmp = {}
        for i, num in enumerate(nums):
            val = target - num
            if num in tmp.keys():
                return [i, tmp[num]]
            else:
                tmp[val] = i
        return []

File: 1/Solution.java
import java.util.HashMap;

class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> tmp = new HashMap<>();
        for (int i = 0; i < nums.length; i++) {
            int num = nums[i];
            int val = target - num;
            if (tmp.containsKey(num)) {
                return new int[]{tmp.get(num), i};
            } else {
                tmp.put(val, i);
            }
        }
        return new int[]{};
    }
}